packages/card/event-card/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Card from '../card'
+import { CardProps, ProductBadgesProps } from '../types'
+import s from './style.module.css'
+
+interface EventCardProps {
+  appearance?: CardProps['appearance']
+  link: string
+  eventDate: string
+  eventType: string
+  title: string
+  productBadges?: ProductBadgesProps['badges']
+}
+
+export function EventCard({
+  appearance,
+  link,
+  eventDate,
+  eventType,
+  title,
+  productBadges,
+}: EventCardProps) {
+  const ctaText = `Register for ${eventType.toLowerCase()}`
+
+  return (
+    <Card heading={title} link={link} appearance={appearance} withArrow={false}>
+      <div className={s.eventCardInner}>
```

Comment:
Can we remove this wrapping div? No CSS is being applied here and removes the ability to use flexbox auto margins on children.

Reply:
Yep that's an artifact of some earlier attempts at making a wrapper to get hover state from. 🔪 shortly!